recipe/build.sh

```diff
@@ -1,8 +1,15 @@
 #!/bin/bash
 
+source activate "${CONDA_DEFAULT_ENV}"
+
 chmod +x configure
 
-./configure --prefix=$PREFIX --enable-cxx
+if [ "$(uname)" == "Darwin" ];
+then
+    ./configure --prefix=$PREFIX --enable-cxx --build=x86_64-apple-darwin
```

Comment:
I see. So this is more about supporting 64-bit and 32-bit or at least that is what this [thread]( https://gmplib.org/list-archives/gmp-bugs/2007-August/000812.html ) concludes. Normally, we don't support 32-bit binaries on Mac or Linux, but if it is helpful to you to have them on Linux I suppose we can try it.